docs/systemd.md

```diff
@@ -0,0 +1,55 @@
+---
+id: systemd
+title: systemd configuration
+---
+
+To manage starting, logging, etc. of Gotify, systemd can be used.
+
+# Installation
+
+For this example, Gotify is installed by root in a directory called
+`/opt/gotify/` and the executable in there is called `gotify`. The config file
+will be `/etc/gotify/config.yml` using relative paths as in the example and the
+service will be run as root. A file called `/opt/gotify.service` should be
+created by root containing:
+
+```desktop
+[Unit]
+Description=Gotify
+Requires=network.target
+After=network.target
+
+[Service]
+Type=simple
+User=root
+WorkingDirectory=/opt/gotify
+ExecStart=/opt/gotify/gotify
+StandardOutput=file:/var/log/gotify/gotify.log
+StandardError=file:/var/log/gotify/gotify-error.log
+Restart=always
+RestartSec=3
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Then the following commands enable systemd to start Gotify at boot:
+
+```bash
+sudo mkdir /var/log/gotify
+sudo chmod -R go-rw /opt/gotify /etc/gotify/config.yml /var/log/gotify
+sudo cd /etc/systemd/system
+sudo ln -s /opt/gotify/gotify.service
```

Comment:
I'm not that fluent with creating custom systemd jobs, but why do we have to create the gotify.service file in the opt folder and then symlink it to /etc/systemd/system, why don't we create it directly in /etc/systemd/system?

If we need this symlink please merge these two commands into one
```
ln -s /opt/gotify/gotify.service /etc/systemd/system/gotify.service
```

Reply:
Yes, symlinking is practiced often and makes updates easier. Commands merged in next commit.